Dockerfile

```diff
@@ -40,7 +40,6 @@ RUN \
   echo "*** Cleanup ***" && \
   mv "./docker/docker-entrypoint.sh" "./" && \
   rm -rf "./docker" && \
-  rm -rf "./.git" && \
```

Comment:
Docker image consists of immutable layers, so this was merely hiding the files without recovering any space.